todo/src/App.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoList from './components/TodoList';
+import { connect } from 'react-redux'
+import { addTask, removeTask, toggleTask } from './actions'
+import TodoForm from './components/TodoForm';
+import styled from 'styled-components';
+
+const Div = styled.div`
+  width: 30vw;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 10px;
+`
+
+
+class App extends Component {
+
+  render() {
+    return (
+      <Div className="App">
+        <TodoForm addTask={this.props.addTask} />
+        <TodoList
+          tasks={this.props.tasks}
+          toggleTask={this.props.toggleTask}
+          removeTask={this.props.removeTask} />
+      </Div>
+    );
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    tasks: state.tasks
+  }
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    addTask: (id) => dispatch(addTask(id)),
+    removeTask: (task) => dispatch(removeTask(task)),
+    toggleTask: (task) => dispatch(toggleTask(task))
```

Comment:
This looks repetitive since you're just taking in a parameter and passing it to your actions; Why not just use an object?
```javascript
const mapDispatchToProps = {
    addTask,
    removeTask,
    toggleTask
};
```